config/metrics.js

```diff
@@ -113,6 +113,10 @@ module.exports = {
 		extract: (item) => !!getBPRules(item)?.allowsDeletions,
 		permissions: ['ADMIN'],
 	},
+	AllowsSecurity: {
+		extract: (item) => !!getBPRules(item)?.AllowsSecurity,
```

Comment:
can you link to the docs for the `AllowsSecurity` field? it's strange that it's capitalized.